releases/v1.4.0-beta.toml

```diff
@@ -0,0 +1,72 @@
+# commit to be tagged for new release
+commit = "HEAD"
+
+project_name = "containerd"
+github_repo = "containerd/containerd"
+match_deps = "^github.com/(containerd/[a-zA-Z0-9-]+)$"
+
+# previous release
+previous = "v1.3.0"
+
+pre_release = true
+
+preface = """\
+The fifth major release of containerd includes a mix of new features and
+expanded support, such as support for CGroups v2, expanded SELinux support,
+support for Windows on Kubernetes through CRI, and support for snapshotters
+based on shared remote storage. All significant bug and stability fixes included
+in this release are also available on supported prior releases. Like previous
+major releases, what is not included in this release is also important. There
+are only two small additions to the API with no backwards incompatible changes,
+allowing client and daemon upgrades to be done independently without disruption.
+The incredible increase in usage of containerd over the last year has shown the
+value of our core principles to easily expand support for different use cases
+while driving toward ever increasing stability.
+
+### Runtime
+* **Support cgroups v2** (https://github.com/containerd/containerd/pull/3799)
+* **Improved SELinux support**
+* **Rework shim logger shutdown process** (https://github.com/containerd/containerd/pull/4162)
+
+### Snapshots
+* **Support target snapshot option on prepare to allow skipping prepare when snapshot reports target already exists** (https://github.com/containerd/containerd/pull/3793)
+* **Snapshotter options passed to backend** (https://github.com/containerd/containerd/pull/4080)
+* **Add filters to walk function** (https://github.com/containerd/containerd/pull/3709)
+* **Support for FUSE mounts** (https://github.com/containerd/containerd/pull/3765)
```

Comment:
Support for lazy-pulling snapshotters like [stargz-snapshotter](https://github.com/containerd/stargz-snapshotter)